clang-tools-extra/test/clang-doc/single-source-html.cpp

```diff
@@ -0,0 +1,8 @@
+// RUN: rm -rf %t
+// RUN: mkdir %t
+// RUN: echo "" > %t/compile_flags.txt
+// RUN: cp "%s" "%t/test.cpp"
+// RUN: clang-doc --format=html --executor=standalone -p %t %t/test.cpp -output=%t/docs > %t/output.txt
```

Comment:
```suggestion
// RUN: clang-doc --format=html --executor=standalone %s -output=%t/docs | FileCheck %s
```
This is the standard way to write tests w/ check lines that come from `stdout`. You may also need `2>&1` before the `\` if you're checking `stderr`